pinc/theme.inc

```diff
@@ -119,30 +119,96 @@ function html_logobar()
 {
     global $code_url, $site_name;
     global $maintenance, $testing, $alert_message;
+    global $blog_url, $wiki_url;
 
     echo "\n<div id='header'>\n";
-    echo "<a href='$code_url'>\n";
+
     echo "<div id='logo'>\n";
-    echo "<img src='$code_url/graphics/dp-logo.svg' alt='$site_name'>\n";
-    echo "</div>\n";
+    echo "<a href='$code_url'>\n";
+    echo "<img id='logo-img' src='$code_url/graphics/dp-logo.svg' alt='$site_name'>\n";
     echo "</a>\n";
+    echo "</div>\n"; // logo
 
-    echo "<div id='titles-preserved'><p>\n";
-
-    // show number of books posted to PG
-    // (takes into account that some books are processed as several projects)
-    echo "<span id='x-preserved'>";
-    echo sprintf(
-        _('%s titles preserved for the world!'),
-        number_format(total_completed_projects())
-    );
-    echo "</span>\n";
-    echo "<span id='month-preserved-summary'>";
-    show_completed_projects(true);
-    echo "</span>";
+    echo "<div id='logo-right'>";
+    $user = User::load_current();
+    if (!$user) {
+        // show number of books posted to PG
+        // (takes into account that some books are processed as several projects)
+        echo "<div id='titles-preserved'>\n";
+        echo "<span id='x-preserved'>";
+        echo sprintf(
+            _('%s titles preserved for the world!'),
+            number_format(total_completed_projects())
+        );
+        echo "</span>\n";
+        echo "<span id='month-preserved-summary'>";
+        show_completed_projects(true);
+        echo "</span>";
+        echo "</div>\n";
+    } else {
+        $icon_links = [];
+        if (get_url_for_forum()) {
+            $icon_links[] = [
+                "name" => _("Inbox"),
+                "class" => "fas fa-envelope fa-2x",
```

Comment:
Perhaps pull "fa-2x" out as something that's common to all the icons?

Reply:
Fair question, but to me the whole class attribute feels like a unit here and I'm wary of breaking that into two pieces, one of which is stashed out in a stylesheet.